sklearn/manifold/_mds.py

```diff
@@ -272,6 +273,67 @@ def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
         return best_pos, best_stress
 
 
+def svd_scaler(dissimilarities, n_components=2):
+    """
+    Computes multidimensional scaling using SVD algorithm
+
+    Parameters
+    ----------
+    dissimilarities : ndarray, shape (n_samples, n_samples)
+        Pairwise dissimilarities between the points. Must be euclidean.
+    n_components : int, optional, default: 2
+        Number of dimension in which to immerse the dissimilarities.
+
+    Returns
+    ----------
+    X : ndarray, shape (n_samples, n_components)
+        Coordinates of the points in a ``n_components``-space.
+
+    stress : float
+        The final value of the stress (sum of squared distance of the
+        disparities and the distances for all constrained points).
+
+    References
+    ----------
+    "An Introduction to MDS" Florian Wickelmaier
+    Sound Quality Research Unit, Aalborg University, Denmark (2003)
+
+    "Multidimensional Scaling" Chapman Hall
+    2nd edition, Boca Raton (2001)
+
+    """
+
+    dissimilarities = check_symmetric(dissimilarities, raise_exception=True)
+
+    n_samples = dissimilarities.shape[0]
+
+    # Centering matrix
+    H = np.eye(*dissimilarities.shape) - (1. / n_samples) * \
+        np.ones(dissimilarities.shape)
+
+    # Double centered matrix
+    K = -0.5 * np.dot(H, np.dot(dissimilarities ** 2, H))
+
+    w, V = linalg.eigh(K, check_finite=False)
+
+    # ``dissimilarities`` is Euclidean iff ``K`` is positive semi-definite.
+    # For detail see "Multidimensional Scaling" Chapman Hall p 397
+    if not np.all(w >= -1e-8):
+        raise ValueError("Dissimilarity matrix must be euclidean")
```

Comment:
We should try to explain how the user might fix this. E.g.

"Make sure to pass an euclidean matrix, or use dissimilarity='euclidean'"

Reply:
Done